theSrc/scripts/PictographConfig.js

```diff
@@ -480,6 +481,11 @@ class PictographConfig {
     }
     this.gridInfo.flexible.row = (_.findIndex(this.gridInfo.sizes.row, { flexible: true }) !== -1)
 
+    const maxGutterHeight = Math.floor(this.size.container.height / (this.gridInfo.dimensions.row - 1))
+    if (this.size.gutter.row > maxGutterHeight) {
```

Comment:
With InsufficientContainerSizeErrors, we silently fail because it should be clear to the user that the output area is too small. However it seems this error (`this.size.gutter.row > maxGutterHeight`) can occur when rowGutterLength is too large or there are too many rows. If we silently fail the user is not able to deduce the issue (they aren't going to check the console). Can we show a simple text message in the output instead? I know this was done before for InsufficientContainerSizeErrors and then removed because users couldn't read the message when the output was too small, but you could reuse that code (check the commit history).

Reply:
So, show the error message as red text on the widget?

Reply:
Yeah, I think that should be simple and communicate the issue to the user.

Reply:
Seems like there is a class for that but it is currently unused: https://github.com/Displayr/rhtmlPictographs/blob/master/theSrc/scripts/DisplayError.js